docker/setup.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+cd /home/beakerx/beakerx
+
+conda create -y -n beakerx python=3.5 jupyter pandas
+source activate beakerx
+./gradlew --no-daemon build
```

Comment:
--no-daemon not needed.
gradlew not needed since gradle was installed directly.
either remove install of gradle or use it.